source/Sashimi/AzurePowerShellActionHandler.cs

```diff
@@ -15,12 +15,20 @@ public class AzurePowerShellActionHandler : IActionHandlerWithAccount
         public bool WhenInAChildStepRunInTheContextOfTheTargetMachine => false;
         public bool CanRunOnDeploymentTarget => false;
         public ActionHandlerCategory[] Categories => new[] { ActionHandlerCategory.BuiltInStep, AzureConstants.AzureActionHandlerCategory, ActionHandlerCategory.Script };
-        public string[] StepBasedVariableNameForAccountIds { get; } = {SpecialVariables.Action.Azure.AccountId};
+        public string[] StepBasedVariableNameForAccountIds { get; } = { SpecialVariables.Action.Azure.AccountId };
 
         public IActionHandlerResult Execute(IActionHandlerContext context, ITaskLog taskLog)
         {
             var syntax = context.Variables.GetEnum(KnownVariables.Action.Script.Syntax, ScriptSyntax.PowerShell);
 
+            var useBundledTooling = context.Variables.GetFlag(KnownVariables.Action.UseBundledTooling, true);
+
+            if (useBundledTooling)
+            {
+                // Warn that the use of bundled tooling is not recommended
+                taskLog.Warn($"Using the Azure tools bundled with Octopus is not recommended. Learn more about Azure Tools: https://g.octopushq.com/AzureTools.");
```

Comment:
The link does not seem to work

Reply:
Yes we are still working through what documentation is required, I expect we won't merge this until that is sorted, although I'm not 100% sure on this process.

Reply:
I've added the link to Octopurls pointing at existing Azure Powershell documentation, will update link if required later